### PR TITLE
Create Test.java

### PR DESCRIPTION
**Pull request by The-Hungg**
Test again.

### CODE REVIEW

File: Test.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Xin chao
```
Comment (Vdhiepp) [inline Test.java:1]: hi

File: Test.java [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Hello
```
Comment (Vdhiepp) [inline Test.java:2]: bye

File: Test.java (hunk 1 above)
Comment (The-Hungg) [reply Test.java:1]: Hello

File: Test.java (hunk 2 above)
Comment (The-Hungg) [reply Test.java:2]: ok

**Pull request merged**